articles/systems/03-boundary-system.md

```diff
@@ -1,23 +1,30 @@
-# Headline
+# The Boundary System
 
-A brief overview of the tutorial.
+The boundary system provides a solution specific boundary implementation for use in your projects, this takes a feed from the vendor supplied data for the boundary and allows you to apply custom textures / details for your project.
 
-## What you'll need
+## Mixed Reality Root configuration system
 
-List previous steps if needed and a short list of required materials or skills.
+In the Root configuration, the specific implementation for the boundary system is selected, which defaults to the XRTK implementation for the boundary system.  In most cases this does not need to be touched, however, advanced users can replace it with their own system if they wish.
 
-## Getting Started
+![](/images/Configuration/BoundarySystem/BoundarySystemProfile.png)
 
-1. Step 1
-2. Step 2
+## Boundary System platform configuration
 
----
+The Boundary system for the XRTK allows for different implementations to be utilized for different runtime or build platforms. Where an interface in to the Vendor implementation for a boundary exists, the XRTK will provide an adapter/provider, however, not all platforms do.
+Where a boundary provider is available, you can customize how the XRTK manages the boundary area as you wish:
+
+![](/images/Configuration/BoundarySystem/BoundarySystemSettings.png)
+
+The configuration holds:
+
+* Display settings - Options to show the boundary (or not), as well as its height, physics layer and default material.
+* Floor settings - Options to show the floor (or not), as well as the material to be used for the floor.
+* Wall settings - Options to show the boundary walls (or not), as well as the material to be used for the walls.
+* Ceiling settings - Options to show the ceiling (or not), as well as the material to be used for the ceiling.
 
-### Related Articles
+## Further notes
 
-* [Article 1]()
```

Comment:
I'd like to see links back to articles that reference this one (like getting started?)